src/player/PlayerComponent.cpp

```diff
@@ -282,6 +297,13 @@ void PlayerComponent::queueMedia(const QString& url, const QVariantMap& options,
   extraArgs.insert("ad", "");
   extraArgs.insert("vd", "");
 
+  extraArgs.insert("tls-verify", "yes");
+  extraArgs.insert("stream-lavf-o", "verifyhost=" + host);
+
+#ifdef Q_OS_LINUX
+  extraArgs.insert("tls-ca-file", "/etc/ssl/cert.pem");
+#endif
```

Comment:
Ah, I looked up what Qt does and it searches a series of dirs; we could do something similar (and could do it once globally at startup).